src/Illuminate/Auth/Access/Gate.php

```diff
@@ -148,6 +150,38 @@ public function denies($ability, $arguments = [])
      */
     public function check($ability, $arguments = [])
```

Comment:
Could this method be simplified as `return $this->getGrant($ability, $arguments)->allowed();` ?